### PR TITLE
ci: fix workflow syntax

### PR DESCRIPTION
**Pull request by sean-krail**
Fix indentation in GitHub Actions workflow files and added `charts/agent-group/values.schema.json`.

### CODE REVIEW

**Comment by sean-krail** [comment]
Wanted to add linting/testing with [helm/chart-testing-action](https://github.com/helm/chart-testing-action), but going to hold off on that until later.

**Review by amilbeck — DISMISSED** [review]

File: charts/agent-group/values.schema.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                },
+                "retryIntervalSeconds": {
+                  "minimum": 5,
+                  "type": "integer",
+                  "description": "Number of seconds it will wait before retrying. This number doubles every retry to a max of 10 minutes (defaults to 5, valid range 5-600)",
+                  "maximum": 600
+                },
+                "agentGroupId": {
+                  "type": "number",
+                  "description": "Agent group Id from Management Console (hover over an agent group to see its Id)"
```
Comment (jj-jitterbit) [inline charts/agent-group/values.schema.json]: "description": "Agent group ID from Management Console (hover over an agent group to see its ID)"

File: charts/agent-group/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                },
+                "retryCount": {
+                  "minimum": 0,
+                  "type": "integer",
+                  "description": "Number of retries if agent is having issues making the call to Harmony cloud for registration (defaults to 10, valid range is 0-300)",
```
Comment (jj-jitterbit) [inline charts/agent-group/values.schema.json:204]: "description": "Number of retries if agent is having issues making the call to the Jitterbit Harmony Cloud for registration (defaults to 10, valid range is 0-300)",

File: charts/agent-group/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+                  "description": "Prefix for each auto-registered agent's name"
+                },
+                "deregisterAgentOnDrainstop": {
+                  "type": "boolean",
+                  "description": "Performs agent deregistration on drainstop/JVM shutdown (defaults to false)"
+                },
+                "retryIntervalSeconds": {
+                  "minimum": 5,
+                  "type": "integer",
+                  "description": "Number of seconds it will wait before retrying. This number doubles every retry to a max of 10 minutes (defaults to 5, valid range 5-600)",
```
Comment (jj-jitterbit) [inline charts/agent-group/values.schema.json]: "description": "Number of seconds the agent will wait before retrying. This number doubles every retry to a maximum of 600 seconds (10 minutes). Defaults to 5, valid range 5-600."

File: charts/agent-group/values.schema.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+          "description": "Agent group's jitterbit.conf file"
+        }
+      }
+    },
+    "affinity": {
+      "description": "Pod affinity and anti-affinity definitions",
+      "$ref": "#/definitions/affinity"
+    },
+    "env": {
+      "description": "List of extra environment variables that may be specified in the container",
```
Comment (jj-jitterbit) [inline charts/agent-group/values.schema.json]: "description": "List of additional environment variables that may be specified in the container",

**Review by jj-jitterbit — DISMISSED** [review]
I've edited the language a bit to make it consistent with other docs. Looks good.

**Review by hiradp — DISMISSED** [review]

**Review by tmercswims — APPROVED** [review]

**Review by hiradp — APPROVED** [review]

**Review by jj-jitterbit — APPROVED** [review]

**Pull request merged**